front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -221,6 +236,38 @@ function BoxesActionsAndTable({
         selectedBoxes={selectedBoxes}
         key="remove-boxes"
       />,
+      <Button
+        onClick={() => {
+          exportBoxData();
+        }}
+        key="export"
+        data-testid="export-button"
+      >
+        <CSVLink
+          data={exportData}
+          filename="Stock Overview"
+          headers={[
+            { label: "Box Number", key: "labelIdentifier" },
+            { label: "Product", key: "product" },
+            { label: "Gender", key: "gender" },
+            { label: "Size", key: "size" },
+            { label: "Number of items", key: "numberOfItems" },
+            { label: "State", key: "state" },
+            { label: "Location", key: "location" },
+            { label: "Tags", key: "tags" },
+            { label: "Shipment", key: "shipment" },
+            { label: "Comment", key: "comment" },
+            { label: "Age", key: "age" },
+            { label: "Last Modified", key: "lastModified" },
```

Comment:
I followed the discussion on the trello card, but shouldn't there be a "Created On" column also? That'd be relevant if `lastModified` is acutally different from null

Reply:
In https://github.com/boxwise/boxtribute/blob/master/front/src/views/Boxes/components/transformers.ts#L23 there's a check for `createdOn`, which is why I didn't include it in the column. Leaving this discussion open in case it is needed.